### DISCUSSION TITLE
Why canvas?

### DISCUSSION THREAD

**Discussion by anovazzi1** [glideapps/glide-data-grid#551] [Ideas]
why use canvas api and not a canvas based api like konvaJS or fabricjs?

**Comment by jassmith** [comment]
Kanva and Fabric both don't give me the flexibility I needed to make it as fast as possible. Working directly with drawing APIs is something of a career long torture I seem to get subjected to, so I just did what made sense to me. I've seen other libs that use those APIs, they always end up paying through the nose for them.

**Reply by anovazzi1** [reply]
understood, make sense, thanks
